devtools/bom-descriptor-json/pom.xml

```diff
@@ -1752,7 +1752,7 @@
                 </dependency>
                 <dependency>
                     <groupId>io.quarkus</groupId>
-                    <artifactId>quarkus-rest-client-microprofile</artifactId>
+                    <artifactId>quarkus-rest-client-mutiny</artifactId>
```

Comment:
Are you sure this is correct? If so, why?

Reply:
Basically I have the same question in every `mutiny` related change :)

Reply:
what happened here :D thanks! fixing

Reply:
oh, it's the dependencies something script. It changed the order of dependencies.

For a moment I didn't know what's going on :)

Reply:
The realc hange is `quarkus-rest-client-microprofile` -> `quarkus-rest-client-reactive`

Reply:
That's what I though :)

Reply:
Thanks for figuring it out, it baffled me :)

Reply:
> oh, it's the dependencies something script

It's magic! 😆 
Someone wanted it to be sorted alphabetically, which makes sense, but also leads to something like this...